Assets/Scripts/SoundManager.cs

```diff
@@ -5,11 +5,11 @@ public class SoundManager : MonoBehaviour {
 
     //crincle sound found here: http://freesound.org/people/volivieri/sounds/37171/
 
-    //public AudioClip crincleAudioClip;
-    public AudioSource crincle;
+    public AudioClip crincleAudioClip;
```

Comment:
does this need to be public? is this property accessed by any class outside of SoundManager? if not, we should make it a `private` `SerializedField` ([reference documentation](https://docs.unity3d.com/ScriptReference/SerializeField.html))